### ISSUE TITLE
0.6.1 Additions Report

### ISSUE THREAD

**Issue by HyperOG** [CloudStrifera/MaCalc#9]
Bug Report:

In Additions:
GPA section in Grade overview
Letter in Grade overview
Category score
icon 

Bugs found in this category: 0

External Bug found: 1

Bug description: Has a bug where the extention layout dosen't keep selected selection 
![image](https://github.com/CloudStrifera/MaCalc/assets/146312508/ac810548-2406-42a7-b66b-f6d455284c82)

![image](https://github.com/CloudStrifera/MaCalc/assets/146312508/e12199b1-640b-4c4f-af68-0c1a3783ff2a)

![image](https://github.com/CloudStrifera/MaCalc/assets/146312508/7dc1ca55-9806-4e4f-a0fc-4e8bfc88dbfa)

![image](https://github.com/CloudStrifera/MaCalc/assets/146312508/9b697cd3-1124-4207-8a81-5c42dd67071d)

Basically, it keeps the selection and showes the selction on the progress page however, dosent't show it on the layout menu

**Comment by Aunuxx** [comment]
the bug should be fixed, check it out on the master branch, if it works then this issue can be closed
